input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
+Compliance refers to a resource meeting all the requirements, constraints, and standards specified in a particular FHIR profile or an implementation guide. These requirements can include mandatory elements, cardinality constraints, data types, terminology bindings, and extensions. A resource may be compliant with multiple implementation guides. For example, in addition to AU Core profiles, a FHIR resource may be compliant with profiles published in project implementation guides, other national implementation guides, or international implementation guides.
```

Comment:
Beginning of comparison page needs an intro bit. Steal from the relationships the paragraph starting from "AU Core aligns to, and leverages, ..." and then adjust for this page.

Reply:
Added intro paragraph.

Reply:
Needs to be adjusted. Do the following, **and then more**:
- remove gender harmony ig
- don't say 'this page'
- merge the three paras into one that makes sense and is about this page. for example the 2nd para should actually be part of introducing the three IGs and therefore could be part of the 2nd sentence **when adjusted**

Reply:
updated